agent/engine/engine_sudo_linux_integ_test.go

```diff
@@ -16,12 +16,24 @@
 package engine
 
 import (
+	"encoding/json"
```

Comment:
nit: can you group the imports to be similar to what they were before: misc packages, empty line, "github.com/aws/amazon-ecs-agent/xxx" packages, empty line, "github.com/other/xxx" packages

Reply:
I did a goimport command and this is how it did it